src/platform/windows/display_base.cpp

```diff
@@ -291,27 +291,51 @@ int display_base_t::init(int framerate, const std::string &display_name) {
   }
 
   //FIXME: Duplicate output on RX580 in combination with DOOM (2016) --> BSOD
```

Comment:
I noticed this comment from loki. My host GPU is a newer AMD GPU  (RX 6600), but perhaps this is related to my problem; DOOM 2016 runs in Vulkan, so it could be using  `DXGI_FORMAT_R8G8B8A8_UNORM` just like DX11 titles in true fullscreen exclusive mode.

I've been testing a game (Dying Light) with "disable fullscreen optimizations" checked. Before this PR, it wasn't possible to run in true exclusive fullscreen; now it works. However, if you toggle between resolutions in the settings menu, eventually it will either cause a driver timeout or complete BSOD. I was able to reproduce the crash on both `libx264` and `amdvce`, and it looks like #605 doesn't solve my issue (but I realize that it wasn't the focus).